### PR TITLE
Added some small improvements

### PR DESCRIPTION
**Pull request by dxd1**

### CODE REVIEW

File: serverless/service/types.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def camel(self):
 
     @property
     def pascal(self):
-        return stringcase.pascalcase(self.identifier)
+        return stringcase.pascalcase(self.identifier.replace("-", "_"))
```
Comment (dxd1) [inline serverless/service/types.py:51]: I had an issue with this function turning `task-servcie` into `Taservice`

Comment (mprzytulski) [reply serverless/service/types.py:51]: there is a `snakecase` which will do what you need

File: serverless/aws/iam/dynamodb.py [hunk 2]
```diff
@@ -6,8 +6,11 @@
 class DynamoDBReader(IAMPreset):
     resource: Table
 
-    def apply(self, service):
+    def apply(self, service, sid=None):
```
Comment (mprzytulski) [inline serverless/aws/iam/dynamodb.py:9]: can we do the same thing for `deny`

File: serverless/service/types.py (hunk 1 above)
Comment (mprzytulski) [reply serverless/service/types.py:51]: where you would like to use `snakecase`?

**Review by mprzytulski — CHANGES_REQUESTED** [review]

**Review by mprzytulski — APPROVED** [review]

**Pull request merged**